src/rgw/rgw_cache.cc

```diff
@@ -135,8 +134,6 @@ void ObjectCache::put(string& name, ObjectCacheInfo& info, rgw_cache_entry_info
   ObjectCacheEntry& entry = iter->second;
   ObjectCacheInfo& target = entry.info;
 
-  invalidate_lru(entry);
-
```

Comment:
separately, it looks like the removal of this call is causing some failures in s3tests - added back in https://github.com/ceph/ceph/pull/19768